timepiece/entries/forms.py

```diff
@@ -4,15 +4,21 @@
 from django import forms
 from django.db.models import Q
 
+from selectable import forms as selectable
 
 from timepiece import utils
 from timepiece.crm.models import Project, ProjectRelationship
 from timepiece.entries.models import Entry, Location, ProjectHours
+from timepiece.entries.lookups import ActivityLookup
 from timepiece.forms import (
     INPUT_FORMATS, TimepieceSplitDateTimeField, TimepieceDateInput)
 
 
 class ClockInForm(forms.ModelForm):
+    activity = selectable.AutoCompleteSelectField(
```

Comment:
FYI: Moving to this widget will require some additional CSS tweaks for the dropdown button

https://django-selectable.readthedocs.org/en/version-0.7.0/advanced.html#using-with-twitter-bootstrap